plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -25,12 +26,14 @@ const renderContent = ( selectedTab?: string ): JSX.Element => {
 
 export default function Content( props: ContentProps ): JSX.Element {
 	const { selectedTab } = props;
+
 	return (
 		<>
 			<div className="woocommerce-marketplace__content">
 				{ renderContent( selectedTab ) }
 			</div>
 			<Footer />
+			<FeedbackModal />
```

Comment:
🗒️  **No props?** I've tried to keep this new component as self-contained as possible, for now. If it needs to be re-used, it can always be expanded later.